main.tex

```diff
@@ -1134,14 +1142,14 @@ \subsubsection*{Beyond individuals: A family of \wnt{} simulation models}
 When looking at the same graph using a circular layout, we observe four clusters of two or more studies, as shown in \nameref{S1_FigClusters}.
 We have also colored the studies according to additional pathways they include and observe that the clusters separate the studies depending on these additional cellular mechanisms.
 The central cluster includes the \wnt{} model by \lee{}~\cite{Lee2003} as well as the studies~\cite{Cho2006, vanLeeuwen2007, Wawra2007, Mirams2010}.
-A second cluster forms around the simulation studies \cite{Mazemondet2012, Kogan2012, Haack2015, Haack2020, Staehlke2020} and either includes the same wet-lab data from~\cite{Bafico2001, Hannoush2008} or the Cell cycle~\cite{Mazemondet2012} or ROS~\cite{Haack2015, Staehlke2020}.
-A third cluster includes the pathways of Notch~\cite{Rodriguez2007, Wang2013} or Notch + MAPK/ERK \cite{Goldbeter2008}. Even though the algorithms locate \cite{Kruger2004} in the same cluster, it is content-wise rather part of the central cluster.
-A forth cluster forms around studies including MAPK/ERK~\cite{Kim2007} or MAPK/ERK + PI3K/Akt~\cite{Padala2017}.
+A second cluster forms around the simulation studies \cite{Mazemondet2012, Kogan2012, Haack2015, Haack2020, Staehlke2020} and either includes the same wet-lab data from~\cite{Bafico2001, Hannoush2008}, the Cell cycle~\cite{Mazemondet2012} or ROS~\cite{Haack2015, Staehlke2020}.
```

Comment:
I think there should be a comma here rather than having two "or"s? Unless I misread